### PR TITLE
Google People (Contact) - Create Contact

### LINKED ISSUE #4969: [ACTION] Google People (Contact) - Create Contact
**Is there a specific app this action is for?**
Google People (Contact) - Create Contact

**Please provide a link to the relevant API docs for the specific service / operation.**
https://developers.google.com/people/api/rest/v1/people/createContact

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #4969

### CODE REVIEW

File: components/google_contacts/actions/create-contact/create-contact.mjs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "google_contacts-create-contact",
+  name: "Create Contact",
+  description: "Creates a contact. [See the docs here](https://developers.google.com/people/api/rest/v1/people/createContact)",
+  version: "0.0.2",
```
Comment (lcaresia) [inline components/google_contacts/actions/create-contact/create-contact.mjs:8]:
```suggestion
  version: "0.0.1",
```

**Review by lcaresia — DISMISSED** [review]
LGTM!

**Comment by DilanAthukorala** [comment]
@michelle0927 Can we define props of each section as optional fields? (currently unable to send request without entering values to all fields of a selected section)

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release.

**Review by lcaresia — APPROVED** [review]
LGTM!

**Pull request merged**